usaspending_api/data/def_codes.csv

```diff
@@ -0,0 +1,18 @@
+DEFC,Public Law,Public Law Short Title,Group Name

```

Comment:
This should live in files.usapending.gov/reference_data, I can upload now if that would be helpful.

Reply:
If you don't mind that would be great. I want to also keep a local copy add this command to our `load_reference_data` command for anyone who pulls down the project.

Reply:
I would recommend against leaving a local copy. That tends to lead to stale data and confusion

Reply:
Located here: https://files.usaspending.gov/reference_data/def_codes.csv

Reply:
Removed local and have a default file location set to that URL.